frontend/src/Editor/WidgetManager.jsx

```diff
@@ -101,12 +103,14 @@ export const WidgetManager = function WidgetManager({ componentTypes, zoomLevel,
     ];
     const integrationItems = ['Map'];
     const layoutItems = ['Container', 'Listview', 'Tabs', 'Modal'];
+    const legacyItems = ['ToggleSwitchLegacy', 'DropdownLegacy'];
```

Comment:
Can we move this to a config file inside Widget Manager?
Move all of constants if possible

Reply:
Done